src/overlays/actors/ovl_En_Daiku2/z_en_daiku2.h

```diff
@@ -5,9 +5,40 @@
 
 struct EnDaiku2;
 
+typedef void (*EnDaiku2ActionFunc)(struct EnDaiku2*, GlobalContext*);
+
+typedef struct EnDaiku2Particle {
+    /* 0x00 */ u8 enabled;
```

Comment:
```suggestion
    /* 0x00 */ u8 isEnabled;
```
Bools usually have the `is` prefix

Reply:
None of the other effect structs seem to have this.

Reply:
Interesting. Should be fine either way then